src/collar/oc_dialog.lsl

```diff
@@ -515,10 +515,10 @@ default {
 
     sensor(integer num_detected){
         //get sensot request info from list
-        list lSensorInfo=llList2List(g_lSensorDetails,0,3);
-        g_lSensorDetails=llDeleteSubList(g_lSensorDetails,0,3);
+        list lSensorInfo=llList2List(g_lSensorDetails,0,1);
+        g_lSensorDetails=llDeleteSubList(g_lSensorDetails,0,1);
 
-        list lParams=llParseStringKeepNulls(llList2String(lSensorInfo,2), ["|"], []);
+        list lParams=llParseStringKeepNulls(llList2String(lSensorInfo,0), ["|"], []);
         list lButtons = llParseStringKeepNulls(llList2String(lParams, 3), ["`"], []);
```

Comment:
same as above:
shouldnt this be also llList2String(lParams, _1_) instead of llList2String(lParams, _3_) as lParams is build from g_lSensorDetails which has now 2 instead of previous 4 entires?